documentation/pole-workspace-guide.adoc

```diff
@@ -0,0 +1,449 @@
+
+.Locations with the highest crime rate
+[source,cypher]
+----
+MATCH (l:Location)<-[:OCCURRED_AT]-(:Crime)
+RETURN l.address AS locale, l.postcode AS postcode, count(l) AS total
+ORDER BY count(l) DESC
+LIMIT 20
+----
+
+This query matches locations with crimes returns the `address` and `postcode` properties of the `Location` nodes and counts all non-null occurences of crimes committed at that location and orders the results in descending order.
+The `LIMIT` clause limits the number of results returned.
+
+In the next section, you'll use the newly refactored `point` property and Cypher spatial functions.
+
+== Distance
+
+Instead of multiple addresses, you can find out how much and what types of crime were committed in the vicinity of one address.
+You can pick any address as your starting point, but for this query you will use an address that may sound familiar.
+
+This is a complex task, so you can break it up.
+Start with finding the starting point and give it an alias (`corrie`):
+
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+RETURN l.position AS corrie
+----
+
+The next step is to locate where the crimes were committed and calculate the distance to Coronation Street:
+
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+WITH l.position AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+RETURN x.address as crime_location, point.distance(x.position, corrie) AS distanceFromCoronationSt
+----
+Since you are interested in crimes committed in the _vicinity_ of Coronation Street, it makes sense to filter the results and leave out locations farther than 500 meters.
+Additionally, it would interesting to see what types of crimes they are.
+The next step is therefore to filter out all criminal locations > 500 meters and aggregate the crime types in the return clause:
+
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+WITH l.position AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+WITH x, c, point.distance(x.position, corrie) AS distance
+WHERE distance < 500
+RETURN x.address AS address, count(c) AS crime_total, collect(c.type) AS crime_type, distance
+----
+
+The last things you need to do is to order the criminal addresses by their distance from Coronation Street and return only the ten closest addresses:
+
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+WITH l.position AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+WITH x, c, point.distance(x.position, corrie) AS distance
+WHERE distance < 500
+RETURN x.address AS address, count(c) AS crime_total, collect(c.type) AS crime_type, distance
+ORDER BY distance
+LIMIT 10
+----
+
+Now you have a neat table of locations, how many and what type of crimes committed, and the distance to 1 Coronation Street.
+
+[NOTE]
+.Challenge
+====
+Can you find the address farthest away from 1 Coronation St and find out what types of crimes have been committed there?
```

Comment:
I like the challenge :)

It allows learners to use what they have learned, modify their query and make it their own.